src/components/Steckbrief/FederalStateOverview.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { HelmetSeo } from '~/components/Helmet/HelmetSeo';
+import { Layout } from '~/components/Layout';
+import { StaticImage } from 'gatsby-plugin-image';
+import { MailToButtonLink, TextLink } from '~/components/Links';
+import { RsvTeaserGrid, FederalStateList } from '~/components/Steckbrief';
+import { WindowLocation } from '@reach/router';
+
+type Props = {
+  location: WindowLocation<unknown>;
+  state: string;
+} & (Queries.FederalStateFromIndexQuery | Queries.FederalStateToIndexQuery);
+
+export const FederalStateOverview: React.FC<Props> = ({
+  location,
+  state,
+  from,
+  to,
+}) => {
+  // manualy join data because graphQL has no OR operator
+  const join: { [key: string]: typeof from['nodes'][number] } = {};
+  [...from.nodes, ...to.nodes].forEach((x) => {
+    join[x.jsonId] = x;
+  });
+  const radschnellwege = {
+    nodes: Object.values(join),
+  };
+  return (
+    <Layout location={location}>
+      <HelmetSeo
+        title="Steckbriefe"
+        description="Die einzelnen Radschnellverbindungen in den verschiedenen Bundesländern in Deutschland."
+      />
```

Comment:
Content-Thema, aber Titel und Description sollten Unique sein. Vorschlag:

```suggestion
      <HelmetSeo
        title="Steckbriefe für BUNDESLAND"
        description="Eine Übersicht der ANZAHL Radschnellverbindungen in BUNDESLAND, Deutschland."
      />
```